### DISCUSSION TITLE
[Input] with clearable cleared the content when pressing ESC

### DISCUSSION THREAD

**Discussion by ultimateBusiness** [livewire/flux#248] [Feature Requests]
It would be great if the input-field with "clearable" active, can have the behavior of clearing the content when pressing the Escape-key.

**Comment by ultimateBusiness** [comment]
You willl have the behavior when you use type="search" - but then you have two buttons to clear the content.

**Comment by jeffchown** [comment]
@ultimateBusiness For now, I'm using a workaround to hide the first 'x' put in place by standard HTML.
This way I can combine Flux's `clearable` and `type="search"` to get the best of both worlds.

The following CSS does the trick:

```
/* clears the ‘X’ from Internet Explorer */
input[type=search]::-ms-clear { display: none; width : 0; height: 0; }
input[type=search]::-ms-reveal { display: none; width : 0; height: 0; }
/* clears the ‘X’ from Chrome */
input[type="search"]::-webkit-search-decoration,
input[type="search"]::-webkit-search-cancel-button,
input[type="search"]::-webkit-search-results-button,
input[type="search"]::-webkit-search-results-decoration { display: none; }
```

Found this tip here: https://medium.com/@rion.mrk/how-to-remove-x-icon-from-search-input-field-or-input-type-search-db3c808405fb
